packages/synapse-react-client/stories/stubs/Button/MuiButton.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import { Button } from './MuiButton'
+import { GetAppTwoTone } from '@mui/icons-material'
+
+const meta = {
+  title: 'UI/MUI/Button',
+  component: Button,
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/0oPm5lLSUva8kyfVNMS6FA/Sage-Style-%26-Component-Library?type=design&node-id=3006-20001',
+    },
+  },
```

Comment:
Defined at the component level here, but designs may also be linked at the story level if desired.